### ISSUE TITLE
Database constraints ignored by Ontop CLI

### ISSUE THREAD

**Issue by jindrichmynarz** [ontop/ontop#777]
Hi,

the [`--constraint` parameter](https://github.com/ontop/ontop/blob/2606d30b83fb01269525d1342947e13bc503a04c/client/cli/src/main/java/it/unibz/inf/ontop/cli/OntopMappingOntologyRelatedCommand.java#L38-L41) of [Ontop CLI](https://ontop-vkg.org/guide/cli.html) is ignored in the current `version5` branch.

If I understand [this comment](https://github.com/ontop/ontop/issues/571#issuecomment-1318586660) correctly, the [implicit database constraints](https://github.com/ontop/ontop/wiki/Implicit-database-constraints) are considered a legacy mechanism, so I would expect the `--constraint` parameter to be removed to indicate that it's not supported.

However, if it is expected to be supported, the CLI should use it. For example:

```java
if (constraintFile != null) {
  configBuilder.basicImplicitConstraintFile(constraintFile);
}
```

**Comment by bcogrel** [comment]
Hi @jindrichmynarz, thanks for reporting this issue. 

That's indeed a bug, as we are still supporting this feature even if we recommend using lenses.

Best,
Benjamin

**Comment by bcogrel** [comment]
Closed by #812
